src/app/page.tsx

```diff
@@ -86,11 +84,14 @@ export default function Home() {
           <Stack p={4} gap={2}>
             <Stack direction="row" justifyContent="space-between">
               <Typography variant="h3">Beef List 🥩📝</Typography>
-              <Link href="/beef/new" style={{ textDecoration: "none" }}>
-                <Button variant="contained" color="secondary">
-                  Start beef
-                </Button>
-              </Link>
+              <Stack direction="row" gap={2}>
+                <BeefSortDropdown sortOption={sortOption} setSortOption={setSortOption} />
```

Comment:
Add a title to this so it's clear what it does

Reply:
What do you mean? A title in the UI?

Reply:
Yes, at least a small one like in mui outlined text area https://mui.com/material-ui/react-text-field/#inputs